integrations/categories.yaml

```diff
@@ -0,0 +1,251 @@
+data-collection:
+  - id: containers-and-vms
+    parent: data-collection
+    name: Containers and VMs
+    description: ""
+    most-popular: false
```

Comment:
How did you generate this list? Any reason to not have it nested/without "parent"? This is extremely confusing as it is.

Reply:
I talked the exact same with Austin, we decided it would need to be in the final data regardless, and handling it from the beginning like this is easier for Austin.

As far as how this was made, I got the childTag from the JSON from each entry, kept the unique ones, and then generated the rest.

Essentially I only had `name` from JSON, and made `id` from that.

Reply:
I don't think it is easier or harder. @Ferroin is going to be using python and it is super easy to convert to whatever data structure he wants during processing.

Reply:
Also, I see that the "parent" is always `data-collection` and the top-level key is `data-collection`. Why do we need "parent"? Is it possible to have different values?

Reply:
I mean, I don't have a preference, that's why I asked Austin if he had one, sure we can do this in a million ways, and still be correct

Reply:
> Also, I see that the "parent" is always data-collection and the top-level key is data-collection. Why do we need "parent"? Is it possible to have different values?

That's what I discussed with Austin, he said we should leave it in anyways

Reply:
The structure here is mirroring what the frontend expects, with the intent being getting things working quickly. The schema can be found here: https://github.com/netdata/netdata/blob/master/docs/.templates/integration/schema.json, this file is supposed to mirror the ‘categories’ property of the structure described there.

If we want to use a nested structure with something like a `children` key we can look into doing that eventually, but

Reply:
But it doesn't mirror (e.g. has `most-popular`, `parent` instead of `parent_id`). I just found it strange to have `X` as a top-level key and `X` as a parent.